.gitmodules

```diff
@@ -8,7 +8,7 @@
 [submodule "external/mono"]
     path = external/mono
     url = ../../mono/mono.git
-    branch = 2019-02
+    branch = 2019-02-xcode11
```

Comment:
Are we sure that the changes are not safe / compatible with current stable Xcode ?
Ideally we would not have to periodically update (merge) mono... too easy to forget

Reply:
@spouliot we have a workaround, but I have filled and issue https://github.com/mono/mono/issues/14792 it has already been assigned by @marek-safar (I don't have the super powers to assign it to myself, but is an easy issue to fix).

Reply:
We have the workaround in 2019-02 now and a PR open for the casting fix, so you should be able to switch to 2019-02 again.